containers/app/StorageSpaceStatus.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { CircularProgress, Dropdown, Icon, generateUID, useUser, usePopperAnchor } from 'react-components';
+import humanSize from 'proton-shared/lib/helpers/humanSize';
+
+const StorageSpaceStatus = () => {
+    const [{ MaxSpace, UsedSpace }] = useUser();
+    const [uid] = useState(generateUID('dropdown'));
+    const { anchorRef, isOpen, toggle, close } = usePopperAnchor();
+
+    const usedPercent = Math.round(UsedSpace / MaxSpace);
+    const maxSpaceFormatted = humanSize(MaxSpace);
+    const usedSpaceFormatted = humanSize(UsedSpace);
+
+    return (
+        <>
+            <CircularProgress
+                className="center"
+                progress={usedPercent}
+                aria-describedby={uid}
+                rootRef={anchorRef}
+                onClick={toggle}
+            >
+                <text
+                    x="50%"
+                    y="50%"
+                    fontFamily="Constantia"
+                    textAnchor="middle"
+                    alignmentBaseline="middle"
+                    stroke="#ffffff"
+                >
+                    i
+                </text>
+            </CircularProgress>
+            <span className="center opacity-40 smaller storage">{usedSpaceFormatted}</span>
+            <Dropdown
+                id={uid}
+                isOpen={isOpen}
+                anchorRef={anchorRef}
+                onClose={close}
+                originalPlacement="left-bottom"
+                auto
+            >
+                <div className="dropDown-content p1">
+                    <div className="absolute top-right mt0-5 mr0-5">
+                        <button className="flex flex-items-center">
+                            <Icon name="close" />
+                        </button>
+                    </div>
+                    <div className="flex">
+                        <div className="pr1 flex flex-items-center">
+                            <div className="relative">
+                                <CircularProgress progress={usedPercent} size={100} backgroundColor="#eeeeee" />
```

Comment:
Should we maybe include these colors in the design system and use as classes? I don't particularly care how we use them (js constants, classnames, whatever) but it would be good to keep track of them somehow.

Reply:
I agree, import colors from `design-system` and inject it in the component.

Reply:
Yes, you can use `fill-*` for SVG or `bg-` classes, will be easier also to theme later :) 
https://design-system-beta.netlify.com/colors/

Reply:
If you need other colors, why not, but I prefer using colors defined by designers, even if this looks not perfect. 
I prefer hearing them saying "Let's improve this" instead of "DO NOT INVENT DESIGN PLZ!" :D